doc/installation/index.rst

```diff
@@ -181,7 +191,7 @@ GnuTLS **(>= 2.8)**         libgnutls28-dev  Development files for gnutls.
 
 #. Install compilation tools ::
 
-     $ apt-get install build-essential cmake
+     $ apt-get install build-essential cmake lua5.2-dev
```

Comment:
Same here, I prefer to split installations, build-essential and cmake from one side and lua5.2-dev from the other side.

If lua5.3-dev is available is better to use it. Its internal api is better to bind with for Centreon Broker.
So, maybe you could make two parts, if lua5.3 isn't available and if it is.